.travis.yml

```diff
@@ -93,16 +93,16 @@ matrix:
         - os: linux
           compiler: clang
           env: EXTENDED_TEST="yes" CONFIG_OPTS="no-asm enable-asan enable-rc5 enable-md2 no-shared -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -D__NO_STRING_INLINES -Wno-unused-command-line-argument"
-        #- os: linux
-        #  addons:
-        #      apt:
-        #          packages:
-        #              - gcc-5
-        #              - g++-5
-        #          sources:
-        #              - ubuntu-toolchain-r-test
-        #  compiler: gcc-5
-        #  env: EXTENDED_TEST="yes" CONFIG_OPTS="--debug no-asm enable-ubsan enable-rc5 enable-md2 -DPEDANTIC" OPENSSL_TEST_RAND_ORDER=0
+        - os: linux
+          addons:
+              apt:
+                  packages:
+                      - gcc-5
+                      - g++-5
+                  sources:
+                      - ubuntu-toolchain-r-test
+          compiler: gcc-5
+          env: UBUNTU_GCC_HACK="yes" EXTENDED_TEST="yes" CONFIG_OPTS="--debug no-asm enable-ubsan enable-rc5 enable-md2 -DPEDANTIC" OPENSSL_TEST_RAND_ORDER=0
```

Comment:
There is reason for why EXTENTED_TEST was first in the list. env shows truncated at build page and having EXTENTED_TEST first makes it easier to identify test as extended and assess overall situation from one glance. I mean when pull request build is half through, you can just see "EXTENDED_TEST, the line won't actually run."